### DISCUSSION TITLE
Using "Back to Wiki home" links breaks other relative links in the Table of Contents

### DISCUSSION THREAD

**Discussion by u-RraaLL** [uBlockOrigin/uBlock-issues#1816] [General] [answered]
"Back to Wiki home" links on articles (or the "Wiki Home" link in the sidebar) are breaking* TOC by adding `/` at the end of the address in the address bar, which in turn changes TOC links from `*/wiki/*` to `*/wiki/wiki/*`.

\* After using them.

I'm not sure if that's a config issue for the relative links or a bug in github itself regarding relative links.

Not filing this as an issue because I'd have to lie on the template :man_shrugging:

**Comment by uBlock-user** [comment]
TOC ?

**Reply by u-RraaLL** [reply]
Table of contents. Literally wiki's home. (also in the sidebar)

Only for relative links, though.

https://github.com/gorhill/uBlock/wiki <- works correctly
https://github.com/gorhill/uBlock/wiki/ <- relative links have a double `/wiki` in them.

**Comment by ghost** [comment, marked as answer]
Try now.

https://github.com/uBlockOrigin/uBlock-issues/wiki/_compare/15324d56d7710a3e9ec7a79aedcad33d8b67a557

**Reply by u-RraaLL** [reply]
Works :)

There's a rouge `[` before the link on https://github.com/uBlockOrigin/uBlock-issues/wiki/Dashboard:-My-rules though.

**Reply by ghost** [reply]
Make your edit.
